sokutei.h

```diff
@@ -0,0 +1,72 @@
+#ifndef SOKUTEI_BENCHMARK
+#define SOKUTEI_BENCHMARK
+
+#define MAX_COUNTERS 100
+#define COUNTER_LENGTH 32
+#define INT int
+
+#ifdef SOKUTEI_NO_STDLIB
+#ifndef SOKUTEI_STDLIB
+#define SOKUTEI_STDLIB
```

Comment:
The general idea is there and I like it! However, I think that you put a bit too much of macro magic into it. Let me elaborate.

The desired behavior is as follows. If the user defines the `SOKUTEI_NO_STDLIB` macro, then we make sure not to call any external functions (basically, we run in no external dependency mode).

Therefore, there is no need for the `SOKUTEI_STDLIB` macro, since compiling with dependencies is the default behavior.

I would also consider entirely removing this macro magic from here. Why? Because it is perfectly fine to use our custom `strcmp` and `strcpy` implementations even if we could use `string.h` or `stdio.h`.

Another point, which would, in my opinion, slightly help readability. If you consider the usages of `strcmp`, you can see that we're only using it to check if two strings are equal. So, instead of `strcmp` we could use something like `string_equals` and make the code more readable. What do you think?